data/json/monsters/cyborgs.json

```diff
@@ -132,7 +132,7 @@
     "material": [ "steel" ],
     "speed": 110,
     "symbol": "M",
-    "aggression": -5,
+    "aggression": -10,
```

Comment:
What's your goal with -10 on aggression?  We really need a way for monsters to kill zombies and then go back to ignoring the player without making them ignore hostile actions from the player in my opinion but this may not be related to that need.